lib/src/widgets/detailCardsWidgets/movie_main_info.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import '../../utils/ui_constants.dart';
+
+class PosterTitle extends StatelessWidget {
```

Comment:
this class name is different from the file's name, change the file name to keep consistency, and check if this happens in another file

Reply:
done